app/vmui/packages/vmui/src/components/Views/JsonView/JsonView.tsx

```diff
@@ -12,7 +12,17 @@ export interface JsonViewProps {
 const JsonView: FC<JsonViewProps> = ({ data }) => {
   const copyToClipboard = useCopyToClipboard();
 
-  const formattedJson = useMemo(() => JSON.stringify(data, null, 2), [data]);
+  const formattedJson = useMemo(() => {
+    const space = "  ";
+    const values = data.map(item => {
```

Comment:
nit, data can be a null or empty array.

Reply:
`data` cannot be `null`.
In the case of an empty array, no errors will occur.